.travis.yml

```diff
@@ -6,7 +6,6 @@ python:
    - "3.3"
    - "3.4"
    - "3.5"
-   - "3.6"
```

Comment:
Sadly [pylint doesn't support Python 3.6 yet](https://github.com/PyCQA/pylint/issues/1072) and our Python 3.6 tests fail.